content/integrate/redis-data-integration/release-notes/rdi-1-14-0.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: Redis Data Integration release notes 1.14.0 (August 2025)
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+description: |
+  New RDI API v2 with enhanced pipeline management.
+  Improved Oracle RAC support with configuration scaffolding.
+  Enhanced metrics and monitoring capabilities.
+  Better TLS/mTLS support across components.
+linkTitle: 1.14.0 (August 2025)
+toc: 'true'
+weight: 978
+---
+
+
+RDI’s mission is to help Redis customers sync Redis Enterprise with live data from their slow disk-based databases to:
+
+- Meet the required speed and scale of read queries and provide an excellent and predictable user experience.
+- Save resources and time when building pipelines and coding data transformations.
+- Reduce the total cost of ownership by saving money on expensive database read replicas.
+
+RDI keeps the Redis cache up to date with changes in the primary database, using a [_Change Data Capture (CDC)_](https://en.wikipedia.org/wiki/Change_data_capture) mechanism.
+It also lets you _transform_ the data from relational tables into convenient and fast data structures that match your app's requirements. You specify the transformations using a configuration system, so no coding is required.
+
+## What's New in 1.14.0
+
+- Unified MySQL connector now handles both MySQL and MariaDB databases consistently
+- Introduced Google Cloud Spanner support with both Google SQL and PostgreSQL dialect compatibility
```

Comment:
We repeat ourselves here twice.
Should be something like: Google Spanner support using a new Collector:
 - Support for both Google and PostgreSQL dialects
 - Support for multi region deployments